### PR TITLE
DDF-2824 fix for duplicate anchors in documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates documentation to avoid having section headings with duplicate titles.

#### Select relevant component teams: 
@codice/docs 
#### Ask 2 committers to review/merge the PR and tag them here.
@ahoffer
@lessarderic
@mcalcote
@shaundmorris

#### How should this be tested?

review content updates

#### Any background context you want to provide?
Section headers are automatically made into anchors for ease of linking. Duplicate anchors could cause a link to not point to the intended section of the documentation.

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-camel-component.adoc [hunk 1]
```diff
@@ -27,9 +24,8 @@ catalog:framework
 
 ===== Sending Messages to Catalog Framework Endpoint
 
-====== Catalog Framework Producer
-
-In Producer mode, the component provides the ability to provide different inputs and have the Catalog framework perform different operations based upon the header values.  
+.Catalog Framework Producer
+In Producer mode, the component provides the ability to supply different inputs and have the Catalog framework perform different operations based upon the header values.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-camel-component.adoc]: ❔ Not sure if it matters but are we consistent on the capitalization of `framework`? in Catalog framework?

**Comment by pklinef** [comment]
:question: Can we add a test to verify there are no duplicate anchors?  Similar to the broken anchor test.

File: distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-camel-component.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-framework-camel-component.adoc]: good catch. It should be consistent

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
I'm going to leave off updating the tests for the moment. There may be changes soon to the docs build that make it a non-issue
